client/src/store.js

```diff
@@ -184,21 +186,36 @@ export const useStore = defineStore("store", {
       this.displayedList.forEach((breed) => (breed.isActive = isActive));
     },
 
-    setAuth(email, password) {
+    setAuth(email, password, router) {
       this.loginEmail = email;
       this.loginPassword = password;
-      this.isLoggedIn = true;
+
       console.log(this.loginEmail, this.loginPassword);
 
-      return fetch("http://localhost:8080/login/", {
-        method: "POST",
-        headers: {
-          "Content-Type": "application/json",
-        },
-        body: JSON.stringify({
-          email: this.loginEmail,
-          password: this.loginPassword,
-        }),
+      return new Promise((resolve, reject) => {
+        fetch("http://localhost:8080/auth/signup", {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            email: this.loginEmail,
+            password: this.loginPassword,
+          }),
+        })
+          .then((response) => response.json())
+          .then((accessToken) => {
+            this.accessToken = accessToken;
+            if (this.accessToken) {
+              this.isSignedIn = true;
+              router.push({
+                name: "breeds",
+              });
+              return this.isSignedIn;
+            }
+
+            return this.isSignedIn;

```

Comment:
Treutno sem nastavil, tako da, ko vrne tokens se nastavi `signedIn `boolean spremenljivka na `true`, in potem se to dato pošlje v `router.js` file in se uorabi v `beforeEach `navigation guardsu